code/modules/projectiles/updated_projectiles/ammunition.dm

```diff
@@ -355,6 +355,165 @@ Turn() or Shift() as there is virtually no overhead. ~N
 
 
 
+//Deployable ammo box
+
+/obj/item/ammobox
+	name = "M41A Ammo Box"
+	desc = "A large, deployable ammo box."
+	w_class = 5
+	icon = 'icons/obj/items/ammo.dmi'
+	icon_state = "ammobox"
+	item_state = "ammobox"
+	var/base_icon_state = "ammobox"
+	var/magazine_amount = 10
+	var/max_magazine_amount = 10
+	var/max_magazine_rounds = 40
+	var/ammo_type = /datum/ammo/bullet/rifle
+	var/magazine_type = /obj/item/ammo_magazine/rifle
+	var/deployed = 0
```

Comment:
Since this is a boolean var, use `TRUE` and `FALSE` every time it happens, for clarity.